fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -704,6 +756,18 @@ def fixup(p: Parameter, is_sharded: bool, size: torch.Size) -> Parameter:
         del self.orig_sizes
         self._reset_lazy_init()
 
+    def parameters(self, recurse: bool = True) -> Iterator[Parameter]:
+        """Returns an iterator over the module parameters, yielding all the parameters
+        part of the model.
+        """
+        # TODO(anj): Use `copy_into_tensor` in order to provide a copy of the
+        # parameters and not the actual parameters. Ideally we don't users to operate on
+        # actual params.
+        if self.ssd_offload and self.ssd_buffer.storage_state == StorageState.ON_DISK:
+            self.ssd_buffer.from_disk(self.buffer_size)
+
+        return super().parameters(recurse=recurse)
```

Comment:
can this just call named_parameters and just remove the names from the results using an transformation function?

Reply:
The base class does the same thing so would prefer deferring to it. let me know if there is a reason why we should not.